editor/editor_fonts.cpp

```diff
@@ -243,10 +243,10 @@ void editor_register_fonts(Ref<Theme> p_theme) {
 	p_theme->set_font("title", "EditorFonts", df_title);
 
 	// Documentation fonts
-	MAKE_DEFAULT_FONT(df_doc, int(EDITOR_GET("text_editor/help/help_font_size")) * EDSCALE);
-	MAKE_BOLD_FONT(df_doc_bold, int(EDITOR_GET("text_editor/help/help_font_size")) * EDSCALE);
-	MAKE_BOLD_FONT(df_doc_title, int(EDITOR_GET("text_editor/help/help_title_font_size")) * EDSCALE);
-	MAKE_SOURCE_FONT(df_doc_code, int(EDITOR_GET("text_editor/help/help_source_font_size")) * EDSCALE);
+	MAKE_DEFAULT_FONT(df_doc, (default_font_size + 1) * EDSCALE);
+	MAKE_BOLD_FONT(df_doc_bold, (default_font_size + 1) * EDSCALE);
+	MAKE_BOLD_FONT(df_doc_title, (2 * default_font_size - 1) * EDSCALE);
```

Comment:
Won't this set the title font size to 27 where it was previously 23? 27 may be a bit too large.

Other than that, this PR looks good to me now.